db/migrate/20190711085633_drop_pgde_course.rb

```diff
@@ -0,0 +1,9 @@
+class DropPGDECourse < ActiveRecord::Migration[5.2]
+  def up
+    drop_table :pgde_course
+  end
+
+  def down
+    # There is no going back as it was legacy.
```

Comment:
Wouldn't it make sense to just have the table there, at least? In case we have to rollback?

Reply:
Just discussed and @defong tells me that there's data that is lost in the drop table. So maybe we should remove the `def down` so that this migration can't be rolled back, or if that doesn't work find a way to raise an error to block this from going backwards (since we wouldn't be able to run the migrations again if we did).

Reply:
done

```ruby

raise ActiveRecord::IrreversibleMigration, "explain"

```